.github/workflows/IPSWDownloads.yml

```
@@ -3,71 +3,82 @@ on:
+    steps:
+      - uses: actions/checkout@v4
+      - uses: brightdigit/swift-build@v1.3.1
+      - uses: sersoft-gmbh/swift-coverage-action@v4
+        id: coverage-files
+        with:          
+          fail-on-empty-output: true
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v4       
+        with:
+          fail_ci_if_error: true
+          flags: swift-${{ matrix.swift-version }},ubuntu
```

actionlint (1.7.7)

property "swift-version" is not defined in object type {os: string; swift: {nightly: bool; version: number}}

(expression)